RunoPW/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+  "author": {
+    "name": "め Kerem",
+    "id": "665482610859966464"
+  },
+  "contributors": [{
+    "name": "Runo",
+    "id": "702935358395908168"
+  }],
+  "service": "Runo",
+  "description": {
+    "en": "Make Friends & Chat With Millions In A Virtual World",
+    "tr": "Müthiş İnsanlarla Dolu İlginç Bir Yer"
+  },
+  "url": "runo.pw",
+  "version": "1.0.0",
+  "logo": "https://imgur.com/utrvCGI",
+  "thumbnail": "https://imgur.com/utrvCGI",

```

Comment:
actually upload a thumbnail or take a screenshot of the website
and it should be i.imgur links ending with .png/.jpg/etc.